spid_cie_oidc/onboarding/forms.py

```diff
@@ -0,0 +1,29 @@
+from django import forms
+from django.utils.translation import gettext_lazy as _
+
+
+class OnboardingRegistrationForm(forms.Form):
+    
+    organization_name = forms.CharField(
+        label= _("organization Name"),
+        error_messages={'required' : _('Enter your organization name')}
+    )
+
+    url_entity = forms.URLField(
+        initial='https://',
+        label=_("url of the entity"),
+        error_messages={'required' : _('Enter your url of the entity')}
+    )
+
+    url_available = forms.URLField(
```

Comment:
please rename to `authn_buttons_page_url`

Reply:
done here
https://github.com/peppelinux/spid-cie-oidc-django/pull/12/commits/b5d533597dbb577007b30f45eed951a3474330a1

thank you